tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md

```diff
@@ -27,4 +27,4 @@ Other free text and markdown formatting can be used elsewhere in the document if
 
 ### [PROPFIND to sub-folder of a shared resources with same name gives 404](https://github.com/owncloud/ocis/issues/3859)
 
-- [webUISharingAcceptShares/acceptShares.feature:73](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature#L73)
+- [webUISharingAcceptShares/acceptShares.feature:16](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature#L16)
```

Comment:
This test will be rewritten and deleted here: https://github.com/owncloud/web/pull/10488